gguf-py/gguf/vocab.py

```diff
@@ -47,15 +52,49 @@ def add_to_gguf(self, gw: GGUFWriter, quiet: bool = False) -> None:
             if not quiet:
                 print(f'gguf: Setting special token type {typ} to {tokid}')
             handler(tokid)
+        for typ, add in self.add_special_token.items():
+            if not quiet:
+                print(f'gguf: Setting add special token type {typ} to {add}')
+            gw.add_bool(f'tokenizer.ggml.add_{typ}_token', add)
```

Comment:
I think the format string should be moved to constants.py, and the keys should added to the GGUF spec.

Reply:
I'm definitely more than happy to do that, I just didn't want to unilaterally make changes to the GGUF format.

Reply:
It's weird we end up with `add_add_bos_token`, `add_add_eos_token` but it doesn't seem like there's a good alternative.